content/en/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions.md

```diff
@@ -347,6 +347,63 @@ kubectl create -f my-crontab.yaml
 crontab "my-new-cron-object" created
 ```
 
+### Publish Validation Schema in OpenAPI v2
+
+{{< feature-state state="alpha" for_kubernetes_version="1.14" >}}
+
+Starting with Kubernetes 1.14, [custom resource validation schema](#validation) can be published as part
+of [OpenAPI v2 spec](docs/concepts/overview/kubernetes-api.md#openapi-and-swagger-definitions) from
+Kubernetes API server.
+
+The published schema will be consumed by [kubectl](/docs/reference/kubectl/overview) to perform client-side
+validation (`kubectl create` and `kubectl apply`) and schema explanation (`kubectl explain`) on custom resources,
```

Comment:
updated

I left the last part unchanged ("The published schema can be consumed for other purposes") because it's not limited to only be used by `kubectl`. For example we know that `server-side apply` is planning to use the schema in future